keycloak/keycloak-admin-ui
# Fixes SAML provider AuthnContext Constraints fields

## Motivation
Fixes https://github.com/keycloak/keycloak-admin-ui/issues/1253

## Brief Description
AuthnContext ClassRefs & DeclRefs were originally created as text fields, they needed to be changed to multiline inputs.  Also cleaned up the id providers tests and fixed a random missing id issue while writing the tests for the above.

## Verification Steps
1. Go to Identify Providers and create a new SAML provider.
2. Verify that you can add multiple values for AuthnContext ClassRefs and AuthnContext DeclRefs using the traditional multiline input components, and that values are saved correctly. 

## Checklist:

## Additional Notes
All identity provider tests passed locally.
